feature-libs/product-configurator/rulebased/components/attribute/types/checkbox-list/configurator-attribute-checkbox-list.component.ts

```diff
@@ -22,13 +23,20 @@ export class ConfiguratorAttributeCheckBoxListComponent
   implements OnInit {
   attributeCheckBoxForms = new Array<FormControl>();
 
+  /**
+   * @deprecated since 4.1: remove redundant input parameter
+   */
   @Input() group: string;
 
   constructor(
+    /**
+     * @deprecated since 4.1: remove ConfiguratorStorefrontUtilsService dependency
+     */
     protected configUtilsService: ConfiguratorStorefrontUtilsService,
-    protected quantityService: ConfiguratorAttributeQuantityService
+    protected quantityService: ConfiguratorAttributeQuantityService,
```

Comment:
can use the parent's constructor, right?

Reply:
I cannot remove the constructor because we need to inject `ConfiguratorStorefrontUtilsService`. This service will be deprecated in the major release and the constructor will be removed until then the constructor should stay.